python/hail/methods/statgen.py

```diff
@@ -255,17 +255,19 @@ def linreg(dataset, ys, x, covariates=[], root='linreg', block_size=16):
     With the default root, the following row-indexed fields are added. The
     indexing of the array fields corresponds to that of ``ys``.
 
-    - **linreg.nCompleteSamples** (*Int32*) -- number of columns used
-    - **linreg.AC** (*Float64*) -- sum of input values ``x``
-    - **linreg.ytx** (*Array[Float64]*) -- array of dot products of each
-      response vector ``y`` with the input vector ``x``
-    - **linreg.beta** (*Array[Float64]*) -- array of fit effect coefficients of
-      ``x``, :math:`\hat\\beta_1` below
-    - **linreg.se** (*Array[Float64]*) -- array of estimated standard errors,
-      :math:`\widehat{\mathrm{se}}_1`
-    - **linreg.tstat** (*Array[Float64]*) -- array of :math:`t`-statistics,
-      equal to :math:`\hat\\beta_1 / \widehat{\mathrm{se}}_1`
-    - **linreg.pval** (*Array[Float64]*) -- array of :math:`p`-values
+    - **linreg.nCompleteSamples** (:py:data:`.tint32`) -- Number of columns used
+    - **linreg.AC** (:py:data:`.tfloat64`) -- sum of input values ``x``.
```

Comment:
These should either all start with caps and have a period at the end, or none of them should. I don't know which one is more consistent.